docs/crio.8.md

```diff
@@ -328,7 +329,9 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--runroot**="": The CRI-O state directory (default: /run/containers/storage)
 
-**--runtimes**="": OCI runtimes, format is runtime_name:runtime_path:runtime_root:runtime_type:privileged_without_host_devices:runtime_config_path (default: [])
+**--runtime-storage**="": Storage for the runtime, format is runtime_name:storage_driver:opt1:opt2...:optN (default: [])
+
+**--runtimes**="": OCI runtimes, format is runtime_name:runtime_path:runtime_root:runtime_type:privileged_without_host_devices:runtime_config_path:storage_driver=opt1=value,opt2=value (default: [])
```

Comment:
A little more verbosity for the descriptions would be good, along with an example or two would be helpful.  For instance, runtime_type, what are the valid values?

Reply:
@TomSweeneyRedHat thanks for the review! Do you mean adding some examples under `--runtime-storage` or `--runtimes` flag? At least the `runtime_type` you are mentioning isn't a new flag. But I just realized that new option  `storage_driver=opt1=value,opt2=value` in the runtime is useless, so the `--runtimes` will remain as it is.